### PR TITLE
tools/get_release_contributors.py: Clarify docs and ignore ImgBotApp & actions-bot

### PR DESCRIPTION
**Pull request by hoffie**
Updates the `tools/get_release_contributors.py` release helper script:
- to clarify usage (we missed the website contributors in the first run)
- update usage for the new website process (no more translate* branches)
- to ignore two more bot users which should not be mentioned.

### CODE REVIEW

File: tools/get_release_contributors.py [hunk 1]
```diff
@@ -5,8 +5,15 @@
...
 Usage:
-./tools/get_release_contributors.py --from release --to translate3_7_0 --github-token YOUR_TOKEN --repo /path/to/your/jamuluswebsite
-./tools/get_release_contributors.py --from r3_6_2 --to r3_7_0rc1 --github-token YOUR_TOKEN --repo /path/to/your/jamulus
+
+## Website
+- cd jamuluswebsite
+- git log release # Find commit ID of previous version. Search for RELEASE:
+- /path/to/jamulussoftware/code/tools/get_release_contributors.py --from COMMIT_ID_FROM_PREVIOUS_STEP --to next-release --github-token YOUR_TOKEN --repo .
+
+## Code
```
Comment (ann0see) [inline tools/get_release_contributors.py:14]:
```suggestion
## Code Repo (jamulussoftware/jamulus)
```

File: tools/get_release_contributors.py [hunk 2]
```diff
@@ -5,8 +5,15 @@
 May need a Github Personal access token to avoid hitting rate limits.
 
 Usage:
-./tools/get_release_contributors.py --from release --to translate3_7_0 --github-token YOUR_TOKEN --repo /path/to/your/jamuluswebsite
-./tools/get_release_contributors.py --from r3_6_2 --to r3_7_0rc1 --github-token YOUR_TOKEN --repo /path/to/your/jamulus
+
+## Website
```
Comment (ann0see) [inline tools/get_release_contributors.py:9]:
```suggestion
## Website (jamulussoftware/jamuluswebsite)
```

File: tools/get_release_contributors.py (hunk 1 above)
Comment (hoffie) [reply tools/get_release_contributors.py]: Done.

File: tools/get_release_contributors.py (hunk 2 above)
Comment (hoffie) [reply tools/get_release_contributors.py]: Done.

File: tools/get_release_contributors.py [hunk 3]
```diff
@@ -5,8 +5,15 @@
 May need a Github Personal access token to avoid hitting rate limits.
 
 Usage:
-./tools/get_release_contributors.py --from release --to translate3_7_0 --github-token YOUR_TOKEN --repo /path/to/your/jamuluswebsite
-./tools/get_release_contributors.py --from r3_6_2 --to r3_7_0rc1 --github-token YOUR_TOKEN --repo /path/to/your/jamulus
+
+## Website repo (jamulussoftware/jamuluswebsite)
+- cd jamuluswebsite
+- git log release # Find commit ID of previous version. Search for RELEASE:
```
Comment (pljones) [inline tools/get_release_contributors.py:11]: Using `--graph` might make the log easier (not sure on that).  Also specifying a `--format` that limits the result to one line per commit, so long as the info is sufficient (if you're not generally a git user, you may not be familiar with configuring your got log format). `RELEASE:` should probably be in quotes to show it's a literal to search for.

Comment (hoffie) [reply tools/get_release_contributors.py]: > Using `--graph` might make the log easier (not sure on that).

Hm, tried with `--graph`, not too convinced about the benefits in this case.

> Also specifying a `--format` that limits the result to one line per commit, so long as the info is sufficient (if you're not generally a git user, you may not be familiar with configuring your got log format).

Added `--oneline`

> `RELEASE:` should probably be in quotes to show it's a literal to search for.

Done.

**Review by softins — COMMENTED** [review]
Looks like `actions-user` also needs to be ignored:
```
tony@pi:~/jamulus $ ./tools/get_release_contributors.py --from r3_8_1 --to r3_8_2 --repo .
Code contributors: @actions-user @ann0see @atsampson @BLumia @corrados @DavidSavinkoff @djfun @gilgongo @henkdegroot @hoffie @j-santander @mirabilos @ngocdh @npostavs @pljones @Rob-NY @RobyDati @softins
Application translators: @actions-user @ann0see @BLumia @dzpex @henkdegroot @hoffie @ignotus666 @jose1711 @jujudusud @melcon @pljones @rolamos @SeeLook @softins
```

File: tools/get_release_contributors.py [hunk 4]
```diff
@@ -28,7 +35,7 @@ class UnexpectedGithubStatus(RuntimeError):
 
 
 # List of user names which should be ignored (such as bots):
-ignore_list = ['github-actions[bot]', 'imgbot[bot]']
+ignore_list = ['github-actions[bot]', 'imgbot[bot]', 'actions-bot', 'ImgBotApp']
```
Comment (ann0see) [inline tools/get_release_contributors.py:38]:
```suggestion
ignore_list = ['github-actions[bot]', 'imgbot[bot]', 'actions-bot', 'ImgBotApp', 'actions-user']
```

As mentioned by @softins

**Review by ann0see — COMMENTED** [review]
It's not that much...

File: tools/get_release_contributors.py (hunk 4 above)
Comment (hoffie) [reply tools/get_release_contributors.py]: Thanks @softins and @ann0see. Added.

**Review by softins — APPROVED** [review]

**Pull request merged**